### PR TITLE
Match battle functions and some unused/dead functions

### PR DESCRIPTION
**Pull request by Rainchus**

### CODE REVIEW

File: src/1A5830.c [hunk 1]
```diff
@@ -440,8 +518,82 @@ ApiStatus EnemyFollowupAfflictTarget(ScriptInstance* script, s32 isInitialCall)
...
+    s32 outVar2;
+    s32 outVar1;
+    s32 hitResult;
+    u8 attackStatus;
+    s32 actorID;
+    s32 battleStatusFlags1Temp;
+    s32 battleFlagsModifier;
+
+
+    outVar1 = get_variable(script, *args++);
```
Comment (nanaian) [inline src/1A5830.c:534]: This isn't an out var; just do `actorID = get_variable(...);`

File: src/1A5830.c [hunk 2]
```diff
@@ -440,8 +518,82 @@ ApiStatus EnemyFollowupAfflictTarget(ScriptInstance* script, s32 isInitialCall)
...
+    actorID = outVar1;
+
+    if (outVar1 == ACTOR_SELF) {
+        actorID = script->owner1.enemyID;
+    }
+
+    actor = get_actor(actorID);
+    outVar2 = *args++;
```
Comment (nanaian) [inline src/1A5830.c:542]: Call this `outVar`

File: src/EA6FD0.c [hunk 3]
```diff
@@ -1,3 +1,11 @@
 #include "common.h"
 
-INCLUDE_ASM(s32, "EA6FD0", func_80240000_EA6FD0);
+s32 func_80240000_EA6FD0(void) {
+    s32 temp = dead_get_variable(NULL, SI_SAVE_VAR(0));
+
+    if (temp <= 0x26) {
+        return 0x19010F;
+    } else {
+        return 0x190110;
```
Comment (nanaian) [inline src/EA6FD0.c:9]: Dead ROM addresses here, fun

File: src/crash.c [hunk 4]
```diff
@@ -66,6 +66,20 @@ INCLUDE_ASM(s32, "crash", func_8002C1D4);
 INCLUDE_ASM(s32, "crash", crash_printf_string);
 
 INCLUDE_ASM(s32, "crash", crash_print_fpr);
+/*
+void crash_print_fpr(s32 x, s32 y, s32 regNum, void *addr) {
```
Comment (nanaian) [inline src/crash.c:70]: Put NON_MATCHING here and a reason why it doesnt match

**Review by nanaian — CHANGES_REQUESTED** [review]

File: src/EA6FD0.c (hunk 3 above)
Comment (ethteck) [reply src/EA6FD0.c:9]: rofl oh wow

**Review by ethteck — APPROVED** [review]
LGTM! great work!

**Pull request merged**